hc/accounts/tests/test_profile.py

```diff
@@ -108,3 +120,8 @@ def test_it_shows_badges(self):
         self.assertNotContains(r, "bobs-tag.svg")
 
     ### Test it creates and revokes API key
+    def test_it_creates_and_revokes_api_key(self):
```

Comment:
@Andela-Didacus @daisyndungu We aren't exercising the API in this test. Please see my previous comment on this.